hub.go

```diff
@@ -366,28 +366,6 @@ func (hub *Hub) Flush(timeout time.Duration) bool {
 	return client.Flush(timeout)
 }
 
-// Continue a trace based on HTTP header values. If performance is enabled this
-// returns a SpanOption that can be used to start a transaction, otherwise nil.
-// TODO: remove - moved to tracing.go.
-func (hub *Hub) ContinueTrace(trace, baggage string) (SpanOption, error) {
```

Comment:
Should we call this change in changelog?

Reply:
This was never released, was part of https://github.com/getsentry/sentry-go/pull/833